test/test_rct_manifest_command.py

```diff
@@ -41,10 +41,10 @@
 
 
 def _build_valid_manifest():
-    manifest_zip = six.StringIO()
+    manifest_zip = six.BytesIO()
```

Comment:
I'm just curious. Why did you replace `StringIO` with `BytesIO`?

Reply:
It was necessary to get the test to pass. If you consider that a zip file is handled as binary data, it makes sense.